commands/user.go

```diff
@@ -468,3 +490,106 @@ func verifyUserEmailWithoutTokenCmdF(c client.Client, cmd *cobra.Command, userAr
 	}
 	return nil
 }
+
+func userConvertCmdF(c client.Client, cmd *cobra.Command, userArgs []string) error {
+	toBot, _ := cmd.Flags().GetBool("bot")
+
+	toUser, _ := cmd.Flags().GetBool("user")
+
+	if !(toUser || toBot) {
+		return fmt.Errorf("either %q flag or %q flag should be provided", "user", "bot")
+	}
+
+	if toBot {
+		return convertUserToBot(c, cmd, userArgs)
+	}
+
+	return convertBotToUser(c, cmd, userArgs)
+}
+
+func convertUserToBot(c client.Client, _ *cobra.Command, userArgs []string) error {
+	users := getUsersFromUserArgs(c, userArgs)
+	for _, user := range users {
+		if user == nil {
+			continue
+		}
+		bot, resp := c.ConvertUserToBot(user.Id)
+		if resp.Error != nil {
+			printer.PrintError(resp.Error.Error())
+			continue
+		}
+
+		printer.PrintT("{{.Username}} converted to bot.", bot)
+	}
+	return nil
+}
+
+func convertBotToUser(c client.Client, cmd *cobra.Command, userArgs []string) error {
+	user := getUserFromUserArg(c, userArgs[0])
+	if user == nil {
+		return fmt.Errorf("could not find user by %q", userArgs[0])
+	}
+
+	password, err := cmd.Flags().GetString("password")
+	if err != nil {
+		return err
+	}
+
+	if password == "" {
+		return errors.New("password is required")
+	}
+
+	up := &model.UserPatch{
+		Password: model.NewString(password),
+	}
+
+	username, _ := cmd.Flags().GetString("username")
+	if username == "" {
+		if user.Username == "" {
+			return errors.New("username is empty")
+		}
+	} else {
+		up.Username = model.NewString(username)
+	}
+
+	email, _ := cmd.Flags().GetString("email")
+	if email == "" {
+		if user.Email == "" {
+			return errors.New("email is empty")
+		}
+	} else {
+		up.Email = model.NewString(email)
+	}
+
+	nickname, _ := cmd.Flags().GetString("nickname")
+	if nickname != "" {
+		up.Nickname = model.NewString(nickname)
+	}
+
+	firstname, _ := cmd.Flags().GetString("firstname")
+	if firstname != "" {
+		up.FirstName = model.NewString(firstname)
+	}
+
+	lastname, _ := cmd.Flags().GetString("lastname")
+	if lastname != "" {
+		up.LastName = model.NewString(lastname)
+	}
+
+	locale, _ := cmd.Flags().GetString("locale")
+	if locale != "" {
+		up.Locale = model.NewString(locale)
+	}
+
+	systemAdmin, _ := cmd.Flags().GetBool("system_admin")
+
+	user, resp := c.ConvertBotToUser(user.Id, up, systemAdmin)
+	if resp.Error != nil {
+		printer.PrintError(resp.Error.Error())
+		return resp.Error
```

Comment:
If the error is returned we don't need to print it
```suggestion
		return resp.Error
```